ofrak_core/test_ofrak/components/test_comments.py

```diff
@@ -38,7 +39,10 @@ async def test_adding_comments(executable_resource: Resource):
 # We suppress the function_scoped_fixture health check because the executable_resource fixture
 # doesn't need to be reset between individual runs of hypothesis (since the comment overrides
 # the previous one every time).
-@settings(suppress_health_check=[HealthCheck.function_scoped_fixture])
+@settings(
+    suppress_health_check=[HealthCheck.function_scoped_fixture],
+    deadline=timedelta(seconds=5),
```

Comment:
I'm not sure this is necessary. What this [deadline](https://hypothesis.readthedocs.io/en/latest/settings.html#hypothesis.settings.deadline) is, as I understand, is the time it takes to generate the hypothesis test case. With default settings, this test gets run 200 times per test run, so the increased time delta could lead to long test runs.

I believe if you run tests on your machine hypothesis will generate new tests cases, so the longer you run it the more likely it might generate large `text` strategies. This, anyway, is my hypothesis on what you are observing. What if you try setting `text(max_size=100)` on line 46? That should increase the reliability of this test within the default 200ms time.

Reply:
@whyitfor , no per the link you cited, 

>  a duration [...] that each individual example (i.e. **each time your test function is called**, not the whole decorated test) within a test is not allowed to exceed. Tests which take longer than that may be **converted into errors** 

(emphasis mine). This is effectively a timeout - if for some reason, the test lasts [significantly] longer, it is reported as an error.

Reply:
I see, you're right. I validated this with the following test:

```python
from hypothesis import strategies as st
from hypothesis import given
import time

@given(text=st.text())
def test_deadline(text: str):
    print(text)
    time.sleep(0.25)
    assert text == text
```